.github/workflows/pretest.yml

```diff
@@ -87,6 +88,7 @@ jobs:
         python -Werror::DeprecationWarning -m compileall -f -q cupy cupyx examples tests docs
         pushd docs
         pip install -r requirements.txt
+        pip install -r rocm-requirements.txt
```

Comment:
Also, do we need hipify_torch on doc build?

Reply:
We also do not need hipify_torch for the doc build since it uses stub files as you mentioned. Updated the pretest.yml to reflect this.